src/models/classifiers.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import numpy as np
+from rdkit import Chem
+from sklearn.linear_model import SGDClassifier
+from sklearn.preprocessing import StandardScaler
+from sklearn.pipeline import make_pipeline
+from sklearn.metrics import ConfusionMatrixDisplay, confusion_matrix
+from sklearn.metrics import recall_score
+from sklearn.metrics import precision_score
+from sklearn.metrics import balanced_accuracy_score
+from sklearn.svm import SVC
+import pickle 
+
+df = pd.read_csv('data/interim/jak2_pIC50_data.csv')
+
+def processing(fp_str): 
+    fp_lst = [int(i) for i in fp_str]
+    fp_np = np.asarray(fp_lst)
+    return fp_np
+
+X_train = df[df['dataset_type'] == 'train'].Fingerprint.apply(processing)
+X_test = df[df['dataset_type'] == 'test'].Fingerprint.apply(processing)
+y_train = df[df['dataset_type'] == 'train'].activity
+y_test = df[df['dataset_type'] == 'test'].activity
+
+y_train = y_train.to_numpy()
+y_test = y_test.to_numpy()
+X_train = list(X_train)
+X_test = list(X_test)
+
+def metrics(classifier_name, y_train_predict, y_test_predict, y_train, y_test):
+    
+    cm_train = confusion_matrix(y_true=y_train_predict, y_pred=y_train)
+    cm_test = confusion_matrix(y_true=y_test_predict, y_pred=y_test)
+    
+    def list_view(array):
```

Comment:
Функцию в функции лучше не делать.. Вынеси List_view

Reply:
Можно сделать что-то такое, чтобы сразу в одной функции CM считать и получать список
```suggestion
    def get_confusion_matrix(y_pred, y_true):
        conf_matrix = confusion_matrix(y_true= y_true, y_pred= y_pred)
        return [conf_matrix[0][0], conf_matrix[0][1], conf_matrix[1][0], conf_matrix[1][1]]
```